OWNERS

```diff
@@ -7,3 +7,4 @@ approvers:
 reviewers:
 - kubernetes-csi-reviewers
 - bswartz
+- sunnylovestiramisu
```

Comment:
@sunnylovestiramisu You are already in [kubernetes-csi-reviewers](https://github.com/kubernetes-csi/csi-release-tools/blob/master/KUBERNETES_CSI_OWNERS_ALIASES) :).